static/app/utils/performance/contexts/pageAlert.tsx

```diff
@@ -2,15 +2,23 @@ import React, {createContext, Fragment, useContext, useState} from 'react';
 import {Theme} from '@emotion/react';
 
 import {Alert} from 'sentry/components/alert';
+import {IconClose} from 'sentry/icons';
+import {useLocalStorageState} from 'sentry/utils/useLocalStorageState';
 
 type PageAlertType = keyof Theme['alert'];
 
 export type PageAlertOptions = {
   message: React.ReactNode | undefined;
   type: PageAlertType;
+  dismissId?: string;
```

Comment:
h: Can we make this a enum instead? And make users of this `PageAlert` add to an enum here?

Reply:
done!